plugin.yml

```diff
@@ -14,6 +14,8 @@ configuration:
       type: boolean
     uploader_version:
       type: string
+    pgp_public_key_url:
+      type: string
```

Comment:
Can we set this to the keybase url by default and then you can set it to an empty string in your particular set up?

Reply:
Addressed here: https://github.com/joscha/codecov-buildkite-plugin/pull/28/commits/f69e53c6009e1aedb346796f63387e65559fa35c